pkg/reconciler/instances/ory/test/ory_integration_test.go

```diff
@@ -77,12 +77,75 @@ func TestOryIntegrationProduction(t *testing.T) {
 		setup.ensureSecretIsDeployed(t, credsName)
 	})
 }
+func TestOryIntegrationEvaluation(t *testing.T) {
+	skipTestIfDisabled(t)
 
-func (s *oryTest) getHorizontalPodAutoscaler(t *testing.T, name string) *autoscalingv1.HorizontalPodAutoscaler {
```

Comment:
I'm generally okay with simplifying this function and moving error checking up.
Could you please also refactor the prod test?